lib/report-portal-client.js

```diff
@@ -138,6 +138,7 @@ class RPClient {
                 this.restClient.create(url, launchData, { headers: this.headers })
                     .then((response) => {
                         this.map[tempId].realId = response.id;
+                        helpers.saveLaunchIdToFile(responce.id);
```

Comment:
Make option for this. I don't need such files in my test project workspace.

Reply:
also `responce` => `response`

Reply:
Done